ui/src/app/components/runtime-configuration-panel/disk-selector.tsx

```diff
@@ -37,6 +67,26 @@ export const DiskSelector = ({
     <FlexColumn
       style={{ ...styles.controlSection, gap: '11px', marginTop: '11px' }}
     >
+      {!disableDetachableReason && (
```

Comment:
I don't think this is the right condition to display the warning.  I believe what we actually want is `computeType` === `ComputerType.Standard`